stock_data.py

```diff
@@ -0,0 +1,73 @@
+import os
+import requests
+from datetime import datetime
+from dotenv import load_dotenv
+load_dotenv()
+
+
+class StockManager:
+    """Requests, manages, and formats data received from the TwelveData API"""
+    def __init__(self):
+        self.index_list = ["SPX", "NDX", "DJI"]
+        self.index_data = {}
+
+    def get_index_data(self):
+        """Gets stock index data from TwelveData API and formats into a nested dictionary.
+        Also adds commas/punctuation and rounds to two decimal places."""
+        for index in self.index_list:
+            response = requests.get(
+                url="https://api.twelvedata.com/quote",
```

Comment:
you can set `https://api.twelvedata.com/quote` as a constant too like 

`twelve_data_host = "https://api.twelvedata.com/quote"